scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv.py

```diff
@@ -0,0 +1,261 @@
+                    converters={'FACID': lambda x: str(x)})
+
+new_columns = ['GeoId', 'Date',
+               'CALicensedHealthcareFacilityBedFACID',
+               'CALicensedHealthcareFacilityName',
+               'CALicensedHealthcareFacilityType',
+               'Count_HospitalBed_SkilledNursingBed',
+               'Count_HospitalBed_SpecialTreatmentProgramBed',
+               'Count_HospitalBed_IntermediateCareHabilitativeBed',
+               'Count_HospitalBed_IntermediateCareNursingBed',
+               'Count_HospitalBed_CongregateLivingHealthFacilityBed',
+               'Count_HospitalBed_IntermediateCareBed',
+               'Count_HospitalBed_CoronaryCareBed',
+               'Count_HospitalBed_IntensiveCareBed',
+               'Count_HospitalBed_IntensiveCareNewbornNurseryBed',
+               'Count_HospitalBed_PerinatalBed',
+               'Count_HospitalBed_UnspecifiedGeneralAcuteCareBed',
+               'Count_HospitalBed_BurnBed',
+               'Count_HospitalBed_PediatricBed',
+               'Count_HospitalBed_RenalTransplantBed',
+               'Count_HospitalBed_RehabilitationBed',
+               'Count_HospitalBed_AcuteRespiratoryCareBed',
+               'Count_HospitalBed_AcutePsychiatricCareBed',
+               'Count_HospitalBed_ChemicalDependencyRecoveryBed',
+               'Count_HospitalBed_PediatricIntensiveCareUnitBed',
+               'Count_HospitalBed_LaborAndDeliveryBed',
+               'Count_HospitalBed_IntermediateCareDevelopmentallyDisabledBed',
+               'Count_HospitalBed_PsychiatricHealthBed',
+               'Count_HospitalBed_PediatricDayAndRespiteCareBed',
+               'Count_HospitalBed_DialysisStationsBed',
+               'Count_HospitalBed_CorrectionalTreatmentCenterBed']
+
+new_df = pd.DataFrame(columns=new_columns)
+
+FDR_to_type = {
+    'SKILLED NURSING FACILITY': 'SkilledNursingFacility',
+    'INTERMEDIATE CARE FACILITY-DD/H/N/CN/IID': 'IntermediateCareFacility',
+    'CONGREGATE LIVING HEALTH FACILITY': 'CongregateLivingHealthFacility',
+    'INTERMEDIATE CARE FACILITY': 'IntermediateCareFacility',
+    'HOSPICE FACILITY': 'HospiceFacility',
+    'GENERAL ACUTE CARE HOSPITAL': 'GeneralAcuteCareFacility',
+    'ACUTE PSYCHIATRIC HOSPITAL': 'AcutePsychiatricFacility',
+    'HOSPICE': 'HospiceFacility',
+    'PEDIATRIC DAY HEALTH & RESPITE CARE FACILITY': 'PediatricDayHealthAndRespiteCareFacility',
+    'CHRONIC DIALYSIS CLINIC': 'ChronicDialysisFacility',
+    'CHEMICAL DEPENDENCY RECOVERY HOSPITAL': 'ChemicalDependencyRecoveryFacility',
+    'CORRECTIONAL TREATMENT CENTER': 'CorrectionalTreatmentCenterFacility'
+}
+
+bed_type_dict = {
+    'SKILLED NURSING': 'SkilledNursingBed',
+    'SPECIAL TREATMENT PROGRAM': 'SpecialTreatmentProgramBed',
+    'INTERMEDIATE CARE/DD HABILITATIVE': 'IntermediateCareHabilitativeBed',
+    'INTERMEDIATE CARE/DD NURSING': 'IntermediateCareNursingBed',
```

Comment:
Can you remind me why we don't parse DD into DevelopmentallyDisabled here? (as opposed to IntermediateCareDevelopmentallyDisabledBed, where we do)